source/dhtslib/sam.d

```diff
@@ -178,43 +216,167 @@ class SAMRecord
         {
             s[i] = seq_nt16_str[bam_seqi(seqdata, i)];
         }
-        return s;
+        return s.idup;
+    }
+
+    /// Assigns sequence and resets quality score
+    pragma(inline, true)
+    @property void sequence(string seq)
```

Comment:
likewise here, I think if you take const(char)[] then you could pass either char[] or string to the function automatically

Reply:
Why did you mark this as resolved if signature still takes string?

Reply:
I see that the file I"m looking at is outdated. Not sure how to see latest version.